raiden-dapp/src/router.ts

```diff
@@ -64,3 +66,38 @@ export default new Router({
     }
   ]
 });
+
+/**
+ * Checks whether token network route is accessed and whether
+ * requested token network address is present in store.
+ */
+export const checkTokenNetworkRoute = (
+  name: string,
+  path: string,
+  next: (to?: string | void) => void,
+  tokens: Tokens
+): void => {
+  [RouteNames.SELECT_HUB as String, RouteNames.OPEN_CHANNEL as String].includes(
```

Comment:
I believe almost all other routes should be checked. Only `HOME` and `SELECT_TOKEN` don't have a `:token` param. Also, it's not a good practice to test if you're in one (or some) specific route on a global guard, it'd be clearer if you did a guard as a separate function (to reuse) and applied it as per-route guards, or doing a global guard which would redirect if a `token` param is present but not in the tokens list (independently of which route it is).